src/pages/ServiceDetails/ServiceMetrics.tsx

```diff
@@ -11,6 +11,31 @@ import { authentication } from '../../utils/Authentication';
 import HistogramChart from './HistogramChart';
 import MetricChart from './MetricChart';
 import { Histogram, MetricGroup } from '../../types/Metrics';
+import history from '../../app/History';
+import { Link } from 'react-router-dom';
+import { style } from 'typestyle';
+
+const expandedChartContainerStyle = style({
+  position: 'absolute',
```

Comment:
This has to be an absolute layout?

Reply:
Yes and no. All these styles can be omitted and the chart will look good. However:
* The chart limits itself to 350px height. So, most of the time it wouldn't use the available vertical space. It is too hard (for me) to correctly set the height without the absolute positioning.
* Also, this is a "hack" to place the expanded chart on top of the tabs (and hide the tabs) to gain more space for the expanded chart.

This is why I said that styles are a little hacky. If you don't like it, I'll just remove all these styles and let the chart to limit itself to 350px height, and allowing the tabs to be visible.

Reply:
@israel-hdez yes, I think that would be the better solution. The absolute stuff makes me nervous

Reply:
@israel-hdez, maybe if you provide height as C3 chart params instead of style, you will avoid the 350px max-height. It can be provided as props: http://c3js.org/reference.html#size-height

Reply:
@jotak The problem is that it isn't possible to set the right height without calculating it in js/ts. But, probably, I'll go this way. Let me try some CSS before doing it via javascript - I don't like too much to do in JS what CSS should be doing :)

Reply:
I know exactly what you mean... I already introduced that kind of dubious height calculation here. Hope we can find a better way at some point.